ambry-api/src/main/java/com.github.ambry/account/Container.java

```diff
@@ -65,12 +69,15 @@
   static final String MEDIA_SCAN_DISABLED_KEY = "mediaScanDisabled";
   static final String REPLICATION_POLICY_KEY = "replicationPolicy";
   static final String TTL_REQUIRED_KEY = "ttlRequired";
+  static final String CONTENT_TYPE_WHITELIST_FOR_FILENAMES_ON_DOWNLOAD = "contentTypeWhitelistForFilenamesOnDownload";
```

Comment:
+1

or `allowedContentTypes`

Reply:
I know the string seems overlong but either suggestion won't work since the names recommended seem to suggest that downloads are allowed only for the content types in the whitelist (which is not true).